DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -1113,8 +1113,9 @@ public class DcMsg {
 
         let count = ptrSize.pointee
         let buffer = UnsafeBufferPointer<Int8>(start: ccharPtr, count: count)
+        let data = Data(buffer: buffer)
         dc_str_unref(ccharPtr)
```

Comment:
we should probably use the keyworkd defer here instead. I think generally that would be wise to introduce when unref'ing pointers.

Reply:
hm, if we introduce a `defer` block after setting `ccharPtr` that opens some questions to me - is it guaranteed that that block is not exectuted before? also, we would have two defer blocks, is that an issue?

i am not that used to `defer`, so i've just chosen the thing i know of :)

but if we can answer the question, we can change the code, np - otoh - that kind of code is very rarely touched, so i would also be fine without `defer`.

Reply:
multiple defers are executed in reverse order: https://docs.swift.org/swift-book/ReferenceManual/Statements.html , so that could be answered.

if we now can know that ccharPtr is nil and not garbage in case it is allocated, most questions would be answerd. but still, not sure that function is worth the detailing.

Reply:
i pushed a thing. the `nil`-or-not-`nil` question only pops up when allocation of the ptrSize fails - that's probably pretty unlikely - and i would expect `ccharPtr` to be nil in this case and that's file to be passed to `dc_str_unref()`